src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,43 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_COLLECTION_SIZE = 10;
+    private Object[] keys = new Object[MAX_COLLECTION_SIZE];
+    private Object[] values = new Object[MAX_COLLECTION_SIZE];
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < keys.length; i++) {
+            if (key != null ? key.equals(keys[i]) : key == keys[i]) {
```

Comment:
```suggestion
            if (key == keys[i] || key != null && key.equals(keys(i)) {
```
As for me better to use such a check, it is easier to read, but yours is good too, so it's up to you which to choose